### ISSUE TITLE
Homepage Mentions

### ISSUE THREAD

**Issue by dshanske** [pfefferle/wordpress-webmention#161]
Want to open this to discussion. We now have homepage mentions going to a special page. But...what should we do with them otherwise? How should we display them?

**Comment by pfefferle** [comment]
I am not sure where you want to display the comments other than on the page you‘ve chosen?

**Comment by dshanske** [comment]
How, was the question?

**Comment by dshanske** [comment]
For example, schmarty and aaronpk kept tagging me in posts

**Comment by armingrewe** [comment]
This is a tricky one. I can only answer this personally, not sure to what extent this applies to others. For me what is happening is that others in my niche topic tag me in their posts about my/their niche topic (the Isle of Islay). Right now this just shows up as a generic mention. I would prefer if this mention/tag would show up as a comment with the content of the post (most of them are Twitter posts, with the occasional FB post mixed in) being displayed.
That's my personal preference, I have no idea if/how this would work for others.

**Comment by dshanske** [comment]
It is being tagged by people that prompted the question

**Comment by dshanske** [comment]
I have to analyze this.

**Comment by dshanske** [comment]
It might be a Semantic Linkbacks question. Tagging is a type of post.

**Comment by armingrewe** [comment]
I agree. To expand on my comment for those not aware: My blogs are about the Isle of Islay (I've got a photoblog and a "general" blog). The people tagging me are mainly local folk posting pictures or news about Islay. Right now they're showing up as plain mentions on the respective "about the blog" pages. For me personally I would prefer if they were to show up as "comments" with the (full?) content of their posts (the vast majority are Tweets).
However, I'm aware that this is my personal use case, I don't know if this would work at all for others.

**Comment by dshanske** [comment]
I wonder if anyone would want to optionally turn tags into posts.

**Comment by tw2113** [comment]
This could overall be getting handled by #215 which will add a widget/shortcode/template tag for displaying mentions on a designated homepage.

**Comment by dshanske** [comment]
I want to note some of the ideas in https://indieweb.org/person_mention as an option here. Namely, once we have mf2 and other parsing in here, to add in filters/actions to make it easier to do different things with these types of mentions.

**Comment by snarfed** [comment]
This shipped many years ago, right? Seems like it should be closed?

**Comment by pfefferle** [comment]
👍

**Comment by dshanske** [comment]
This was a question about styling them differently or different display options, not about supporting them.
